HW_3/task_3.py

```diff
@@ -0,0 +1,22 @@
+"""
+Задайте список из вещественных чисел, округленных до сотых.
+Найдите разницу между максимальным и минимальным значением дробной части элементов.
+
+Ввод: значение типа <list> (либо значения типа <int> – размерность списка)
+Вывод: значение типа <float>
+
+Пример:
+[1.1, 1.2, 3.1, 5, 10.01]
+2.0
+"""
+my_list = [1.1, 1.2, 3.1, 5, 10.01]
+my_list_2 = []
+for i in range(len(my_list)):
+    my_list_2.append(my_list[i]-int(my_list[i]))
```

Comment:
Кроме того, дробную участь можно получить отделением на единицу с остатком. Да и функцию `range` использовать необязательно.
```suggestion
for i in my_list:
    my_list_2.append(i % 1)
```